examples/dmd/heat_conduction_dmdc.cpp

```diff
@@ -0,0 +1,717 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//      libROM MFEM Example: DMDc for Heat_Conduction (adapted from ex16p.cpp)
+//
+// Compile with: make heat_conduction_dmdc
+//
+// =================================================================================
+//
+// Sample runs and results for DMD:
+//
+// Command 1:
```

Comment:
Some description of the control would be informative, as well as some explanation of how this example differs from the DMD example in `heat_conduction.cpp`.

Reply:
Done.